### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for `minmax(auto, 1fr)`

### ISSUE THREAD

**Issue by djoume** [stylelint/stylelint#8735]
### What minimal example or steps are needed to reproduce the bug?

```css
.product-gallery {
  grid-template-columns: repeat(auto-fit, minmax(auto, 1fr));
}
```

### What minimal configuration is needed to reproduce the bug?

```js
export default {
  rules: {
  "declaration-property-value-no-unknown": true,
  },
};
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AHATtArmALgWgHMBDAGxJjQE8ACYAHQDtrqC0BLKPHGAWxRKLc8kEph4MAzvGpoYKGIIAURTDgh4AZmxwAaajzYMeRAB7LVEPQEYNaAJR2A3IwC+IHSC3kAckR5xEGBM-fhgkMAkJd3AIBi0CBBB6JmZqOhA0THIJdOlk1IK0kFgwATRBNli8dAh5NBxKPAA3UkwYPAZ1TAYAa06AdwZc6hxMmEYCl1doyDi2AgAxCDRjHESAKwlY6NgUKMR8ookG8hJDHGH0gW5j9J0Jo5OYM4Z8Wfi8Y6IGKCI0KEuIGuMFuIGmLiAA)

### Which Stylelint-related dependencies are you using?

```  
    stylelint:
        specifier: 16.23.1
        version: 16.23.1(typescript@5.9.2)
      stylelint-config-standard:
        specifier: 39.0.0
        version: 39.0.0(stylelint@16.23.1(typescript@5.9.2))
```

### What did you expect to happen?

No error because minmax can be used with auto within a repeat, see https://developer.mozilla.org/en-US/docs/Web/CSS/minmax

### What actually happened?

```
Unexpected unknown value "repeat(auto-fit, minmax(auto, 1fr))" for property "grid-template-columns" (declar
ation-property-value-no-unknown) - declaration-property-value-no-unknown
```

**Comment by romainmenke** [comment]
Hi @djoume,

Do you have an example, spec link, or working demo for `minmax(auto, 1fr)`?
As far as I know and can read in the docs it isn't allowed to combine those.

These are fine however:
- `minmax(0, 1fr)`
- `minmax(auto, 200px)`

But combining `auto` and `fr` dimensions doesn't seem specified as valid.

**Comment by djoume** [comment]
Hi @romainmenke, I couldn't find an example for `minmax(auto, 1fr)`, but I found one for  `minmax(min-content, 1fr)` [here](https://drafts.csswg.org/css-grid-2/#example-337b574e) which also leads to [(declaration-property-value-no-unknown)](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown)

**Comment by romainmenke** [comment]
[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AHATtArmALgWgHMBDAGxJjQE8ACYAHQDtrqC0BLKPHGAWxRKLc8kEph4MAzvGo82DHkQAeAClkNhEBty0AaagEYAZmgCUAbkbNWHLr36CYG0eKnU0MFDEHKATHrUKKmoaWjC6BsYm5pYs7JzcfAJCImKS0u6e3kSYOBB4hmw4-nKBqnIh2kXUPgAMNSiKURZMsTYJ9skQzmluHl44ytm5+YXF8krKNXpGptEt1vF2SY4pLul9WTl5BVUBE8GQoeEzTYwAviA6IAXkAHJEPHCIMIoP-DBIYBISl+CaBQQECB6C1mHQQGhMOQJODpCDmAiEeDYGABGhBGxNHh0BBPGgcJQ8AA3UiYRwMPKYBgAawpAHcGLDqDhITAYswzudfocAQAxCBoBQ4IEAKwkml+sBQP0Q8Oo4IkBPIJDkOCZ4OWivBOhiCqVMBVWhCALwiqIDCgRDQUHVIE1apAXLOQA)

```css
.product-gallery {
  grid-template-columns: minmax(min-content, 1fr);
  grid-template-columns: repeat(2, minmax(min-content, 1fr));
  grid-template-columns: repeat(auto-fit, minmax(min-content, 200px));
  grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
  grid-template-columns: repeat(auto-fit, minmax(min-content, 1fr)); /* only this errors */
}
```

All the specification examples are considered valid and don't produce errors.

It seems that the combination is more complex than I initially thought.

`minmax(min-content, 1fr)` is not valid for the currently specified syntax when used inside `repeat(auto-fit, ...)`

-------

There are some options:
1. Stylelint and the specification are correct and your provided example is not intended to work.
2. The specification is inaccurate or out of date and `repeat(auto-fit, minmax(min-content, 1fr))` is intended to just work.

If you see [this codepen](https://codepen.io/romainmenke/pen/EaVLKNB) it seems that this just isn't correct syntax.

Can you provide an example of where this does work in a browser?

**Comment by djoume** [comment]
@romainmenke  you are correct, `repeat(auto-fit, minmax(min-content, 1fr))` doesn't work, apologies for the distraction.

**Comment by romainmenke** [comment]
No problem! Happy to help :)
